### ISSUE TITLE
Should use training set but not test set to fine tune the models

### ISSUE THREAD

**Issue by realliyifei** [realliyifei/ML-Project-Hotel-Cancellation-Prediction#2]
Basic model experiment section; Cannot recall why used test set for tuning...

**Comment by realliyifei** [comment]
solved.
